app/controllers/schools_controller.rb

```diff
@@ -131,7 +131,7 @@ def individual_student_dashboard_data(student)
       first_name: student.first_name,
       last_name: student.last_name,
       id: student.id,
-      homeroom: student.try(:homeroom).try(:name),
+      homeroom: student.try(:homeroom).try(:educator).try(:full_name),
```

Comment:
This seems like it could easily be misinterpreted.  This seems like a great change, but what do you think about renaming this field to `homeroom_text` to clarify it's not a homeroom, homeroom id, or homeroom name?

Reply:
@kevinrobinson good point. How about homeroom_label?

Reply:
do it! 👍